### DISCUSSION TITLE
TESS systematics - Earth and Moon position

### DISCUSSION THREAD

**Discussion by ghost** [lightkurve/lightkurve#1132] [General]
Hi all,
I am trying to correct the systematics in TESS related to the scattered light coming from Earth and Moon.
For doing that I need to have the angle between my targets (or the center of the focal plane) and the Earth or Moon.
Did anyone know if the ephemeris of Earth and Moon as seen by TESS are stored somewhere, or it is necessary to recalculate them from SPICE kernels ?

**Comment by barentsen** [comment]
I'm not aware of Earth/Moon angles being stored in any public data products, but here are two thoughts:
* An alternative to using SPICE yourself is to retrieve RA/Dec of Earth and Moon from [JPL Horizons](https://ssd.jpl.nasa.gov/horizons.cgi) by entering `@tess` in the "Observer location" field.
* Over the years, I believe that several teams concluded that the pixel data itself tends to be the best predictive variable for modeling systematics (as opposed to any telemetry).  One notable exception where telemetry has proven to be useful is the high-resolution attitude data ("quaternions", e.g. see [Vanderburg et al 2019](https://arxiv.org/pdf/1905.05193.pdf)).

Does that help?
